<!DOCTYPE html>


<html lang="" data-theme="">
<head>
    
        
<meta charset="utf-8">
<meta name="HandheldFriendly" content="True">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="referrer" content="no-referrer-when-downgrade">

<title>Inhee Park, PhD (Computational Protein / Machine Learning Engineer)</title>

<meta name="description" content="SUMMARY Machine Learning Engineer in Biotech MS in Computer Science PhD in Computational Chemical Physics WORK HISTORY Computational Protein / Machine Learning Engineer, Aether Biomachines, Menlo Park, CA [2021-Current]">





<link rel="icon" type="image/x-icon" href="https://iparkirk.github.io/favicon.ico">
<link rel="apple-touch-icon-precomposed" href="https://iparkirk.github.io/favicon.png">


<style>
  body {
    visibility: hidden;
    opacity: 0;
  }
</style>

<noscript>
  <style>
    body {
      visibility: visible;
      opacity: 1;
    }
  </style>
</noscript>



    





    
    
        
    
    

    
        <link rel="stylesheet" href="https://iparkirk.github.io/css/style.min.c6a5c56d82abc1aa3a1925bf0b67884906e878e4e5bfb5af069b0a3e35e32569.css" integrity="sha256-xqXFbYKrwao6GSW/C2eISQboeOTlv7WvBpsKPjXjJWk=">
    





    

    





    
    
        
    
    

    
        <script src="https://iparkirk.github.io/js/script.min.766e48fac645ce89febc4e3a78467b3c44aed2452453b409fa47401ebe483365.js" type="text/javascript" charset="utf-8" integrity="sha256-dm5I&#43;sZFzon&#43;vE46eEZ7PESu0kUkU7QJ&#43;kdAHr5IM2U="></script>
    







<meta property="og:title" content="" />
<meta property="og:description" content="SUMMARY Machine Learning Engineer in Biotech MS in Computer Science PhD in Computational Chemical Physics WORK HISTORY Computational Protein / Machine Learning Engineer, Aether Biomachines, Menlo Park, CA [2021-Current]" />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://iparkirk.github.io/resume/" /><meta property="article:section" content="" />
<meta property="article:published_time" content="2023-05-19T00:00:00+00:00" />
<meta property="article:modified_time" content="2023-05-19T00:00:00+00:00" />


<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content=""/>
<meta name="twitter:description" content="SUMMARY Machine Learning Engineer in Biotech MS in Computer Science PhD in Computational Chemical Physics WORK HISTORY Computational Protein / Machine Learning Engineer, Aether Biomachines, Menlo Park, CA [2021-Current]"/>











    
</head>
<body>
    <a class="skip-main" href="#main">Skip to main content</a>
    <div class="container">
        <header class="common-header"> 
            
                <div class="header-top">
    <h1 class="site-title">
    <a href="/">Inhee Park, PhD (Computational Protein / Machine Learning Engineer)</a>
</h1>
    <ul class="social-icons">


    
        <li>
            <a href="https://www.linkedin.com/in/inheepark" title="Linkedin" rel="me">
            <span class="inline-svg" >




    
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="currentColor" d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"/></svg>

</span>

            </a>
        </li>
    

    
        <li>
            <a href="https://github.com/ipark-CS" title="Github" rel="me">
            <span class="inline-svg" >




    
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><path fill="currentColor" d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>

</span>

            </a>
        </li>
    

    
        <li>
            <a href="https://gitlab.com/ipark" title="Gitlab" rel="me">
                <span class="inline-svg" >




    
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
<g>
	<path fill="currentColor" d="M123.5,52.9c-2.7-7.8-13.8-7.8-16.6,0L57.2,206.6h116C173.1,206.6,123.5,52.9,123.5,52.9z M31.8,283.9
		c-2.3,7,0.3,14.9,6.2,19.3l217.9,161.7L57.1,206.6L31.8,283.9z M173.1,206.6L256,464.9l82.9-258.4H173.1z M480.2,283.9l-25.3-77.3
		L256,464.9l217.9-161.7C479.9,298.7,482.4,290.9,480.2,283.9z M405.2,52.9c-2.7-7.8-13.8-7.8-16.6,0l-49.7,153.6h116L405.2,52.9z"
		/>
</g>
</svg>


</span>

            </a>
        </li>
    




</ul>
</div>

    <nav>
        
        
        <a class="" href="https://iparkirk.github.io/resume/" title="">Resume</a>
        
        <a class="" href="https://iparkirk.github.io/research/" title="">Biotech</a>
        
        <a class="" href="https://iparkirk.github.io/projects/" title="">Computer Science</a>
        
    </nav>




            
        </header>
        <main id="main" tabindex="-1"> 
            
    

    <article class="post h-entry">
        <div class="post-header">
            <header>
                <h1 class="p-name post-title"></h1>

                
            </header>
        </div>
        <div class="content e-content">
            <h3 id="summary" >SUMMARY
<span>
    <a href="#summary">
        <svg viewBox="0 0 28 23" height="100%" width="19" xmlns="http://www.w3.org/2000/svg"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/></svg>
    </a>
</span>
</h3><ul>
<li>Machine Learning Engineer in Biotech</li>
<li>MS in Computer Science</li>
<li>PhD in Computational Chemical Physics</li>
</ul>
<h3 id="work-history" >WORK HISTORY
<span>
    <a href="#work-history">
        <svg viewBox="0 0 28 23" height="100%" width="19" xmlns="http://www.w3.org/2000/svg"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/></svg>
    </a>
</span>
</h3><p>Computational Protein / Machine Learning Engineer, <a href="https://www.aetherbio.com/">Aether Biomachines</a>, Menlo Park, CA [2021-Current]</p>
<ul>
<li>
<p>Machine Learning for Protein Sequence/Structure/Function Prediction &amp; Generation</p>
<ol>
<li><strong>Built content-based recommender system :</strong>
<em>recommended</em> chemical reaction pattern similarity-based protein/enzyme sequences;
<em>expedited</em> protein/enzyme selection period from one month to one hour;
<em>resulted</em> in actuating five new projects in two weeks (Python, Scikit-Learn, Numpy)</li>
<li><strong>Created prompt-engineered sequence generator by Transformer GPT2 :</strong>
<em>trained</em> GPT2 from scratch with BPE tokenizer (chemical reaction and sequence motif-unit tokens);
<em>elicited</em> chemical reaction-prompted, diffused protein/enzyme sequences;
<em>expanded</em> chemical/biological space to explore for protein design without extensive prior structural knowledge
(Pytorch, Hugging Face Transformer, ESMfold)</li>
<li><strong>Implemented human-in-the-loop training strategy for Active Learning :</strong>
<em>improved</em> predictive model accuracy by 10% by
<em>redesigning</em> iterative and interactive training strategy
<em>querying</em> low confidence data first from the unlabeled dataset and
<em>iterative training</em> with updated trainset (Python, Scikit-Learn, ModAL)</li>
</ol>
</li>
<li>
<p>Software Engineering for Computational Structural Biology Tools</p>
<ol>
<li>Structured and integrated chem/bio/ML pipelines/toolchains and deployed them in Docker
(RDkit, CoLab Hallucination/Diffusion, etc)</li>
<li>Deployed end-to-end 3D simulation tool from simple JSON inputs using AmberTools/openMM</li>
<li>Developed automatic catalytic site identification tool in conjunction with ESMFold, AlphaFold2</li>
</ol>
</li>
</ul>
<p>Postdoctoral Fellow (Computational Simulation/Modeling), <a href="https://www.novartis.com/research-development/novartis-institutes-biomedical-research">Novartis Institutes for BioMedical Research</a>, San Diego, CA [2012-2016]</p>
<ul>
<li>
<p><strong>Developed Rational Design Model:</strong>
<em>developed</em> a simple, generic yet robust biological simulator outperforming non-profit software by
<em>accelerating</em> computation time (GPU-enabled simulation),
<em>generating/incorporating</em> ensemble features, and
<em>benchmarking</em> against a wide range of biologics systems;
<em>enabled</em> the rational experimental design to cut time/labor/budget
(Python, R, SQLite, GPU-enabled MD suite compile/simulation)</p>
</li>
<li>
<p><strong>Deployed End-to-End Integrated Tool:</strong>
<em>deployed</em> high-confidence, full de novo sequencing tool for antibody biotherapeutics by
<em>utilizing</em> public database and
<em>reconstructing</em> short tags by Dynamic Programming;
<em>provided fully functional in-house service</em> at no risk of confidential information
disclosure by leveraging combined information protocol (genomics/proteomics/informatics)
(Python, R (ggplot2), Database Retrieval)</p>
</li>
</ul>
<p>Postdoctoral Fellow (Computational Biophysics), <a href="https://www.cityofhope.org/research/beckman-research-institute">Beckman Research Institute at City of Hope</a>, Duarte, CA [2010-2012]</p>
<ul>
<li>Optimized the high-dimensional biological sampling space by refinement protocol,
which was enhanced by reusing the robotics locomotion algorithm used in NASA/JPL.</li>
</ul>
<hr>
<h3 id="education" >EDUCATION
<span>
    <a href="#education">
        <svg viewBox="0 0 28 23" height="100%" width="19" xmlns="http://www.w3.org/2000/svg"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/></svg>
    </a>
</span>
</h3><p>MS. Computer Science (3.95/4.0), San Jose State University, CA, 2020</p>
<ul>
<li>AI-publication: <strong>Multi-Agent Deep Reinforcement Learning for Walker Systems</strong>,
<a href="https://ieeexplore.ieee.org/document/9680106">20th IEEE International Conference on ML and Applications (ICMLA)</a></li>
</ul>
<p>PhD. Chemical Physics (Computational Drug Design), The Ohio State University, OH, USA, 2010</p>
<p>BS. Chemistry and Physics, SookMyung Women&rsquo;s University, Seoul, Korea</p>
<hr>
<h3 id="computer-skills" >COMPUTER SKILLS
<span>
    <a href="#computer-skills">
        <svg viewBox="0 0 28 23" height="100%" width="19" xmlns="http://www.w3.org/2000/svg"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/></svg>
    </a>
</span>
</h3><ul>
<li>Programming Languages &ndash;— [Advanced] Python, Pandas, R, Bash, MySQL and LaTeX &mdash;
[Skilled] Java, C and Verilog with ModelSim</li>
<li>ML / Deep Learning &mdash; Scikit-learn, PyTorch; [Deep Reinforcement Learing] Ray RLlib, OpenAI Gym</li>
<li>Computational Chemistry/Simulation/Modeling Tools — RDkit, OpenMM, Amber, Schrodinger, ChimeraX</li>
<li>Computing Environment &ndash;— Git, Jupyter, Linux, HPC (MPI), GPU Computing, AWS, Conda and Docker</li>
<li>Full Stack Web —&ndash; [Front] HTML, CSS, Bootstrap, Hugo, Javascript/jQuery and Ajax; [Server] JSP/Tomcat</li>
<li>Social Network &ndash; Gephi, iGraph</li>
<li>Computational Simulation &amp; Modeling Tools &mdash; RDkit, OpenMM, Amber, Schrodinger and ChimeraX</li>
</ul>
<hr>
<h3 id="coursework-in-computer-science" >COURSEWORK in COMPUTER SCIENCE
<span>
    <a href="#coursework-in-computer-science">
        <svg viewBox="0 0 28 23" height="100%" width="19" xmlns="http://www.w3.org/2000/svg"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/></svg>
    </a>
</span>
</h3><p>Graduate Courses</p>
<ul>
<li>Design and Analysis of Algorithms</li>
<li>Practical Computer Vision using Convolutional Neural Networks</li>
<li>Cryptography and Computer Security</li>
<li>Topics in Machine Learning</li>
<li>Topics in Wireless Mobile Networking</li>
<li>Web Intelligence (Mining of Massive Dataset)</li>
<li>Social Network Analysis</li>
<li>Applied Probability and Statistics</li>
</ul>
<p>Undergraduate Courses</p>
<ul>
<li>Discrete Mathematics</li>
<li>Introduction to Computer Systems</li>
<li>Data Structures and Algorithms</li>
<li>Operating Systems</li>
<li>Computer Architecture</li>
<li>Object-Oriented Design</li>
<li>Introduction to Database Management Systems</li>
</ul>
<hr>
<h3 id="csml-projects-in-ai-nlp-social-network--webapp" >CS/ML PROJECTS in AI, NLP, SOCIAL NETWORK &amp; WEBAPP
<span>
    <a href="#csml-projects-in-ai-nlp-social-network--webapp">
        <svg viewBox="0 0 28 23" height="100%" width="19" xmlns="http://www.w3.org/2000/svg"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/></svg>
    </a>
</span>
</h3><p>Multi-Agent Deep Reinforcement Learning for Walkers (RayRLlib,Python)</p>
<blockquote>
<p>(1) Found optimal minibatch size and sampling reuse ratio improve performance of PPO algorithm.
(2)  Proposed efficient DRL training strategies of transfer learning and parameter sharing for the robot-legs walkers carrying a package:</p>
</blockquote>
<p>Semantic Textual Similarity using Transfer Learning and Embeddings (TensorFlow,Python)</p>
<blockquote>
<p>Build a Semantic Search Engine as a pragmatic application of Semantic Textual Similarity combined with a faster similarity
search method of approx- imate k-nearest neighbor: <a href="https://bit.ly/2Fjzj4j">https://bit.ly/2Fjzj4j</a></p>
</blockquote>
<p>Sentiment Analysis using Machine Learning and Deep Learning (TensorFlow,Python,C)</p>
<blockquote>
<p>Contrary to expectation, the primitive TF-IDF with Naive Bayes classification (80%)
outperforms the state-of-art contextualized embedding NLP model of BERT (75%) in
capturing the simple polarity of lecture evaluation reviews: <a href="https://bit.ly/37zkpTM">https://bit.ly/37zkpTM</a></p>
</blockquote>
<p>Lighter, Faster Semantic Segmentation by Post-Training Quantization and Quantization-Aware Training (TensorFlow,Python)</p>
<blockquote>
<p>Reduced neural network model-size while retaining inference accuracy us- ing various quantization workflows with MobileNet-based semantic seg- mentation framework of DeepLabV3+: <a href="https://bit.ly/39C1KbM">https://bit.ly/39C1KbM</a></p>
</blockquote>
<p>Deep Reinforcement Learning for Resource Management in 5G (RayRLlib)</p>
<blockquote>
<p>DRL is an intelligent network slicing tool to optimize service allocation by maximize the targeted reward function (spectrum efficiency, quality of ex- perience etc.) <a href="https://bit.ly/380ROtB">https://bit.ly/380ROtB</a></p>
</blockquote>
<p>Power of BTS ARMY for Social Change Envisaged by Twitter Network (Gephi,iGraph,R,Python)</p>
<blockquote>
<p>Graph Theory based structural analysis &amp; Semantic Network based behavioral analysis
for the 24 hours of #MatchAMillion Twitter dataset. Bilingual translators play a crucial role as Local Bridges connecting fandom vs. non- fandom communities: <a href="https://bit.ly/3n4cGp6">https://bit.ly/3n4cGp6</a></p>
</blockquote>
<p>Grocery Nutrient Tracker (GNT)-Market Full Stack WebApp (MySQL, Bootstrap, Java, JSP)</p>
<blockquote>
<p>(1) Build a full-stack WebApp (Bootstrap for front-end, JSP/Tomcat for Web Server,
MySQL for back-end) for convenient and healthy selection for gro- cery list.
(2) Users can generate a personalized grocery list by selecting a dish menu and by
specifying dietary restriction: <a href="https://bit.ly/33XhK7O">https://bit.ly/33XhK7O</a></p>
</blockquote>
<p>Data Compression using Dictionary-based Technique LZW and Its Variant to GIF Image Data Compression (Python)</p>
<blockquote>
<p>Implement the LZW compression algorithm, which is lossless, greedy, and adaptive
dictionary data structure by taking advantage of the frequency redundancy of
character-stream: <a href="https://bit.ly/3n2qlvY">https://bit.ly/3n2qlvY</a></p>
</blockquote>
<p>The BlowFish Block Cipher System – Expensive Key Scheduling (Java)</p>
<blockquote>
<p>Implement one of the Feistel Block Ciphers, BlowFish, using a built-in functionalities
from the cryptoUtil. A larger size of subkey space is time-consuming prior to
encryption, but increases security by making brute-force attempts infeasible:
<a href="https://bit.ly/3hwLU6H">https://bit.ly/3hwLU6H</a></p>
</blockquote>
<hr>
<h3 id="selected-publications" >SELECTED PUBLICATIONS
<span>
    <a href="#selected-publications">
        <svg viewBox="0 0 28 23" height="100%" width="19" xmlns="http://www.w3.org/2000/svg"><path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71" fill="none" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2"/></svg>
    </a>
</span>
</h3><ul>
<li>
<p>S. Kyriacou, G. Meshulam-Simon, L. Clark, C. Chia and Inhee Park,
“Enzyme engineering by means of reaction space indexing coupled with machine learning algorithms”,
<a href="https://www.simbhq.org/annual/">Society for Industrial Microbiology and Biotechnology (SIMB), Aug 2022, San Francisco</a> <br>
<sup>#enzyme engineering, #graph neural network (gnn), #machine learning, #automation
</sup></p>
</li>
<li>
<p>Inhee Park and Teng-Sheng Moh,
“Multi-Agent Deep Reinforcement Learning for Walker Systems”,
<a href="https://ieeexplore.ieee.org/document/9680106">2021 20th IEEE International Conference on Machine Learning and Applications (ICMLA),
2021, pp. 490-495, doi: 10.1109/ICMLA52953.2021.00082.</a>  <br>
<sup>#deep reinforcement learning (drl), #ai, #multi-agent
</sup></p>
</li>
<li>
<p>Inhee Park, J. Venable, C. Steckler, S. Cellitti, S.A. Lesley, G. Spraggon and A. Brock,
“Estimation of Hydrogen Exchange Protection Factors from MD Simulation Based on Amide Hydrogen Bonding Analysis”.
<a href="https://pubs.acs.org/doi/10.1021/acs.jcim.5b00185">J Chem Inf Model. 55(9):1914-25. 2015</a>  <br>
<sup>#monte carlo, #mass spectrometry, #proteomics, #molecular simulation, #computational structural biology
</sup></p>
</li>
<li>
<p>Inhee Park and A. Brock,
“Genomics/Proteomics/Informatics Complementary Approach to High Confident Reconstruction of Antibody V-region Sequence”
<a href="https://sandiegoacs.org/wp-content/uploads/2022/04/april-2015.pdf">ACS West Coast Analytical Chemistry Symposium, Apr 2015, San Diego</a>  <br>
<sup>
#bioinformatics, #genomics, #proteomics, #mass spectrometry, #dynamic programming
</sup></p>
</li>
<li>
<p>S. Bhowmik, D. H. Jones, H.-P. Chiu, Inhee Park et al.
“Structural and Functional Characterization of BaiA, An Enzyme Involved in Secondary Bile Acid Synthesis in Human Gut Microbe”.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23836456">Proteins. 82(2):216- 29. 2014</a>  <br>
<sup>
#quantum chemistry, #catalytic activity
</sup></p>
</li>
<li>
<p>Inhee Park, V. Gangupomu, J. Wagner, A. Jain and N. Vaidehi,
“Structure Refinement of Protein Low Resolution Models Using the GNEIMO Constrained Dynamics Method”.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22260550">J Phys Chem B. 116(8):2365-75. 2012</a>  <br>
<sup>
#homology modeling, #sampling, #molecular simulation, #free energy, #computational structural biology
</sup></p>
</li>
<li>
<p>A. Jain, Inhee Park and N. Vaidehi, “Equipartition Principle for Internal Coordinate Molecular Dynamics”.
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3549275/">J Chem Theory Comput. 8(8):2581-2587. 2012</a>  <br>
<sup>
#free energy
</sup></p>
</li>
<li>
<p>V. Gangupomu, J. Wagner, Inhee Park et al.
“Mapping Conformational Dynamics of Proteins using Torsional Dynamics Simulations”
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23663843">Biophys J. 104(9):1999-2008. 2013</a>  <br>
<sup>
#molecular simulation, #free energy, #computational structural biology
</sup></p>
</li>
<li>
<p>G.S. Balaraman, Inhee Park et al.
“Folding of Small Proteins Using Constrained Molecular Dynamics”
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3124276/">J Phys Chem B. 115(23):7588-96. 2011</a>  <br>
<sup>
#protein folding, #molecular simulation, #free energy, #computational structural biology
</sup></p>
</li>
<li>
<p>W. Harvey, Inhee Park, O. Rubel, V. Pascucci, P.T. Bremer, C. Li and Y. Wang,
“A Collaborative Visual Analytics Suite for Protein Folding Research”
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25068440">J Mol Graph Model. 53:59-71.2014</a>  <br>
<sup>
#graph, #folding, #molecular simulation, #free energy, #computational structural biology
</sup></p>
</li>
<li>
<p>Inhee Park and C. Li,
“Dynamic ligand-induced-fit simulation via enhanced conformational samplings and ensemble dockings: a survivin example”
<a href="https://www.ncbi.nlm.nih.gov/m/pubmed/20337446">J Phys Chem B. 114(15):5144-53. 2010</a>  <br>
<sup>
#induced-fit-docking, #molecular simulation, #free energy, #enhanced sampling
</sup></p>
</li>
<li>
<p>Inhee Park and C. Li,
“Characterization of molecular recognition of STAT3 SH2 domain inhibitors through molecular simulation”
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21360612">J Mol Recognition. 24(2):254-65. 2010</a>  <br>
<sup>
#docking, #molecular simulation, #quantum chemistry, #free energy, #cheminformatics
</sup></p>
</li>
<li>
<p>S. Chattier, J. V. Cooley, Inhee Park et al.
“Design, Synthesis and Biological Studies of Survivin Dimerization Modulators that Prolong Mitotic Cycle”
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23968825">Bioorg Med Chem Lett. 23(19):5429-33. 2013</a>  <br>
<sup>
#in silico design, #docking, #molecular simulation, #cheminformatics, #free energy
</sup></p>
</li>
<li>
<p>Co-authored with 16 authors,
“Impairment of Glioma Stem Cell Survival and Growth by a Novel Inhibitor for Survivin–Ran Protein Complex”
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4295559/">Clin Cancer Res. 19(3):631-42. 2013</a>  <br>
<sup>
#in silico design, #drug repurposing, #translation of breast cancer lead to brain cancer
</sup></p>
</li>
</ul>

        </div>
        

    


<div class="post-info">
    
        <div class="post-date dt-published">2023-05-19</div>
    

    <a class="post-hidden-url u-url" href="https://iparkirk.github.io/resume/">https://iparkirk.github.io/resume/</a>
    <a href="https://iparkirk.github.io/" class="p-name p-author post-hidden-author h-card" rel="me">Inhee Park</a>


    <div class="post-taxonomies">
        
            
    </div>
</div>

    </article>

    

    

    
        







    

        </main>
        
            <footer class="common-footer">
    
    

    <div class="common-footer-bottom">
        
        <div class="copyright">
            <p>© Inhee Park, 2023<br>
            Powered by <a target="_blank" rel="noopener noreferrer" href="https://gohugo.io/">Hugo</a>, theme <a target="_blank" rel="noopener noreferrer" href="https://github.com/mitrichius/hugo-theme-anubis">Anubis</a>.<br>
            
            </p>  
        </div> 

        




<script>
const STORAGE_KEY = 'user-color-scheme'
const defaultTheme = "light-without-switcher"

let currentTheme
let switchButton
let autoDefinedScheme = window.matchMedia('(prefers-color-scheme: dark)')

const autoChangeScheme = e => {
    currentTheme = e.matches ? 'dark' : 'light'
    document.documentElement.setAttribute('data-theme', currentTheme)
    changeButtonText()
}

document.addEventListener('DOMContentLoaded', function() {
    switchButton = document.querySelector('.theme-switcher')
    currentTheme = detectCurrentScheme()
    if (currentTheme == 'dark') {
        document.documentElement.setAttribute('data-theme', 'dark')
    }
    if (currentTheme == 'auto') {
        autoChangeScheme(autoDefinedScheme);
        autoDefinedScheme.addListener(autoChangeScheme);
    }

    if (switchButton) {
        changeButtonText()
        switchButton.addEventListener('click', switchTheme, false)
    }
  
    showContent()
})

function detectCurrentScheme() {
    if (localStorage.getItem(STORAGE_KEY)) {
        return localStorage.getItem(STORAGE_KEY)
    } 
    if (defaultTheme) {
        return defaultTheme
    } 
    if (!window.matchMedia) {
        return 'light'
    } 
    if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        return 'dark'
    }
    return 'light'
}

function changeButtonText()
{   
    if (switchButton) {
        switchButton.textContent = currentTheme == 'dark' ?  "Light theme" : "Dark theme"
    }
}

function switchTheme(e) {
    if (currentTheme == 'dark') {
        localStorage.setItem(STORAGE_KEY, 'light')
        document.documentElement.setAttribute('data-theme', 'light')
        currentTheme = 'light'
    } else {
        localStorage.setItem(STORAGE_KEY, 'dark')
        document.documentElement.setAttribute('data-theme', 'dark')
        currentTheme = 'dark'
    }
    changeButtonText()
}

function showContent() {
    document.body.style.visibility = 'visible';
    document.body.style.opacity = 1;
}
</script>   
    </div>

    <p class="h-card vcard">

    <a href=https://iparkirk.github.io/ class="p-name u-url url fn" rel="me">Inhee Park</a> 

    

    
</p> 
</footer>

        
    </div>
</body>
</html>
